### PR TITLE
pull missing meta information (dependencies, keywords, repo) from latest release

### PR DESCRIPTION
**Pull request by bcoe**
registry-relational stores several pieces of meta-information that the website expects (dependencies, repo, keywords) on the `latest` release stored in the `versions` object.

This pull adds some tests for `maybeUpgradeRRPackageData()`, and populates this additional information:

reviewers: @nexdrew, @aredridel

### CODE REVIEW

**Comment by aredridel** [comment]
This rocks. Sheep when tests pass!

File: test/fixtures/packages/rr_rimraf.json [hunk 1]
```diff
@@ -0,0 +1 @@
...,"2.0.0":"2012-01-30T02:20:45.661Z","2.0.1":"2012-01-30T17:07:51.384Z","2.0.2":"2012-06-08T21:19:19.066Z","2.0.3":"2012-12-15T22:59:57.912Z","2.1.0":"2012-12-20T23:23:50.937Z","2.1.1":"2012-12-21T02:23:38.182Z","2.1.2":"2013-01-22T20:50:45.534Z","2.1.3":"2013-02-06T00:24:04.069Z","2.1.4":"2013-02-06T14:58:43.510Z","2.2.0":"2013-06-21T14:52:14.555Z","2.2.1":"2013-07-14T18:48:37.747Z","2.2.2":"2013-07-22T16:20:26.910Z","2.2.3":"2013-11-29T16:55:07.432Z","2.2.4":"2013-11-29T16:59:28.538Z","2.2.5":"2013-12-09T17:05:49.993Z","2.2.6":"2014-01-15T23:58:31.660Z","2.2.7":"2014-05-05T22:17:46.141Z","2.2.8":"2014-05-06T16:00:05.895Z","2.3.0":"2015-03-03T23:25:07.298Z","2.3.1":"2015-03-05T02:37:34.817Z","2.3.2":"2015-03-10T01:13:34.964Z","2.3.3":"2015-04-30T14:18:05.546Z","2.3.4":"2015-05-19T01:37:39.894Z","2.4.0":"2015-06-07T17:20:09.158Z","2.4.1":"2015-06-30T18:02:24.393Z","2.4.2":"2015-07-19T21:44:13.360Z","2.4.3":"2015-08-26T22:30:40.696Z","2.4.4":"2015-11-20T00:41:43.301Z","created":"2011-02-08T08:55:37.756Z","modified":"2015-11-20T00:41:43.301Z"},"version":"2.4.4","publisher":{"name":"isaacs","email":"i@izs.me","url":null},"lastPublishedAt":"2015-11-20T00:41:43.301Z","dependentCount":3}}
```
Comment (stubbornella) [inline test/fixtures/packages/rr_rimraf.json]: Would it break things to have human readable fixtures?

**Comment by stubbornella** [comment]
Looks good to me!

File: test/fixtures/packages/rr_rimraf.json (hunk 1 above)
Comment (bcoe) [inline test/fixtures/packages/rr_rimraf.json]: nope, we can pretty print the JSON -- I'm quite happy to.

**Pull request merged**